### ISSUE TITLE
`Span` references `LoggingContext` in its docs

### ISSUE THREAD

**Issue by fabianfett** [apple/swift-distributed-tracing#80]
```swift
/// A `Span` represents an interval from the start of an operation to its end, along with additional metadata included
/// with it. A `Span` can be created from a `Baggage` or `LoggingContext` which MAY contain existing span identifiers,
/// in which case this span should be considered as "child" of the previous span.
```

We should remove the ref to `LoggingContext` here.

**Comment by slashmo** [comment]
I'm pretty sire that's not the only reference to the "old" context propagation mechanism. Konrad already started updating the documentation, not sure though if this particular reference was already removed: https://github.com/apple/swift-distributed-tracing/pull/69

**Comment by ktoso** [comment]
Yeah this will be solved by the docs rewrite, ongoing here: https://github.com/apple/swift-distributed-tracing/pull/69

Since you also have a few things we can fix right away perhaps the plan forward is to keep merging and releasing 0.x things until we're happy and you've proven it out in adoption a bit :-)
